src/axom/quest/IntersectionShaper.hpp

```diff
@@ -1075,11 +1077,25 @@ class IntersectionShaper : public Shaper
   // Switch back to public. This is done here because the CUDA compiler
   // does not like the following template functions to be private.
 public:
+  /*!
+   * \brief Set the name of the material used to account for free volume fractions.
+   * \param name The new name of the material.
```

Comment:
Please add a note to the doxygen that `name` cannot contain an underscore

Reply:
Done